Audacia.Mail.MailTrap/MailtrapClient.cs

```diff
@@ -20,8 +20,17 @@ private static SmtpSettings GetSettings(string username, string password)
         /// <param name="username">The username for Mailtrap.</param>
         /// <param name="password">The password for Mailtrap.</param>
         public MailtrapClient(string username, string password)
-			: base(GetSettings(username, password))
+			: base(GetSettings(username, password, HostType.Test))
 		{
+        }
+
+        /// <summary>Initializes a new instance of the <see cref="MailtrapClient"/> class.</summary>
+        /// <param name="username">The username for Mailtrap.</param>
+        /// <param name="password">The password for Mailtrap.</param>
+		/// <param name="hosttype">The host type for Mailtrap.</param>
```

Comment:
nit: minor formatting issue (this is indented too much), and could the parameter be renamed `hostType` please?

Reply:
renamed it to hostType. The indentation looks fine in both VS code and VS.